controllers/datadogagent/feature/livecontainer/feature.go

```diff
@@ -28,26 +33,57 @@ func init() {
 func buildLiveContainerFeature(options *feature.Options) feature.Feature {
 	liveContainerFeat := &liveContainerFeature{}
 
+	if options != nil {
+		liveContainerFeat.runInCoreAgent = options.RunProcessChecksOnCoreAgent
```

Comment:
Do we need to add a version check here as well?

Reply:
I was working under the assumption that the option would only be used internally, however, I think I will move the current version check I have into the `Configure` function after line 77.  That way, we will need less changes down the line.

Reply:
No, we do not. The version check is only necessary when a version is specified in `spec.override`. Otherwise, latest is used.

Reply:
In this case, it seems the version check would need to be a little bit different than the one I have. We would need to check the value of this variable instead of the override.

Reply:
I updated the override function to handle this case.

Reply:
Thanks, change looks good. We check both and override taking precedence which is how overrides work.

Do you think test is warranted for default tag value (might be tricky to write such test)?